src/Settings/components/editBaniOrder.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { ListItem, Avatar } from "@rneui/themed";
+import { useSelector } from "react-redux";
+import PropTypes from "prop-types";
+import { nightModeStyles, nightModeColor } from "../styles/nightModeStyles";
+import STRINGS from "../../common/localization";
+
+function EditBaniOrder({ navigate }) {
+  const { isNightMode } = useSelector((state) => state);
+  const { containerNightStyles } = nightModeStyles(isNightMode);
+  const nightColor = nightModeColor(isNightMode);
+  const { EDIT_BANI_ORDER } = STRINGS;
+  const rearrangeIcon = require("../../../images/rearrangeicon.png");
+  return (
+    <ListItem
+      bottomDivider
+      containerStyle={containerNightStyles}
+      onPress={() => navigate("EditBaniOrder")}
+    >
+      <Avatar source={rearrangeIcon} />
```

Comment:
Missing alt attribute in Avatar component.

Reply:
Veer ji, the Avatar component doesn't include an 'alt' attribute. I can add 'title' if you say so.

Reply:
fixed